.github/workflows/dependency_review.yml

```diff
@@ -0,0 +1,22 @@
+name: dependency_review
+
+on:
+  pull_request:
+    branches:
+      - main
+      - hotfix
+      - feature/**
+
+permissions:
+  contents: read
+
+jobs:
+  dependency-review:
```

Comment:
Why not this https://github.com/aws-amplify/amplify-backend/blob/d1716d174dc77aaee5171efb47318893ac3a9665/.github/workflows/health_checks.yml#L99 ?

Reply:
Since this job doesn't depend on any other steps, it's probably fine as a separate workflow. But yeah, this could also work

Reply:
We can do that too, I assumed every check in `health_checks` follows this https://github.com/aws-amplify/amplify-backend/blob/d1716d174dc77aaee5171efb47318893ac3a9665/.github/workflows/health_checks.yml#L4-L13 as that line seems easy to miss (which is very true in my case since I didn't see that line)